_dashboards/dashboard/index.md

```diff
@@ -0,0 +1,163 @@
+6. From the **New Visualization** window, choose **[eCommerce] Average Sold Quantity**. The donut chart is added to the dashboard.
+
+You've created a basic dashboard like the following, which you'll continue using throughout this tutorial.
+
+![Creating a basic dashboard]({{site.url}}{{site.baseurl}}/images/dashboards/dashboard-basic.png)
+
+## Creating and saving visualizations
+
+Continuing with the dashboard you created in the preceding steps, you'll create a new visualization and save it to the dashboard.
+
+1. From the dashboard toolbar, choose **Create new**.
+1. From the **New Visualization** window, choose **Vertical Bar** and then select the index pattern **opensearch_dashboards_sample_data_ecommerce**. You'll see an image like the following.
+
+    ![New visualization vertical bar chart]({{site.url}}{{site.baseurl}}/images/dashboards/new-bar-viz.png)
+
+1. Configure the chart's **Data** settings by verifying **Metrics** is **Y-axis Count**.
+1. From the **Buckets** panel, choose **Add** and select **X-axis** from the **Add Bucket** dropdown list.
+1. From the **Aggregation** dropdown list, select **Date histogram**.
+1. Choose **Update**. You'll see a bar chart visualization like the following:
+
+    ![Configuring a visualization]({{site.url}}{{site.baseurl}}/images/dashboards/configure-bar-viz.png)
+
+1. From the toolbar, select **Save**
+1. In the **Save visualization** window, add a title for the visualization. For example, the title for the bar chart panel is Order Date.
+1. Select **Save and return**.  
+
+The bar chart visualization is now saved and you are taken back to the dashboard. You'll see three visualizations on the dashboard, as shown in the following image: 
+
+![Dashboard showing visualizations combined in a single view]({{site.url}}{{site.baseurl}}/images/dashboards/new-dashboard.png)
+
+## Adding panels
+
+Continuing with the dashboard you created in the preceding steps, you'll add a visualization to the dashboard.
+
+1. From the dashboard toolbar, choose **Add**.
+1. From the **Add panels** window, choose **[eCommerce] Sales by Category**.
+1. Select `x` to close the **Add panels** window. 
+
+You'll see an area chart visualization display on the dashboard, as shown in the following image: 
+
+![Adding another panel to the dashboard]({{site.url}}{{site.baseurl}}/images/dashboards/new-area-viz.png)
+
+## Editing visualizations
+
+You can edit visualizations, configure panels, and specify labels for a panel. For example, you can change the visualization's colors, title, and other attributes. Visualizations can be edited in the panel itself or in edit mode. Only changes made in edit mode are persistent, that is, they are saved in the visualization itself and will be reflected on any dashboard that uses the visualization. Editing in the panel is helpful for making minor changes at the dashboard level without saving them.
```

Comment:
I think usage of the term "edit mode" here is confusing, particularly because the dashboard itself has an edit mode and a view mode. What we want to distinguish between is tweaking the panel itself vs actually editing the visualization itself. Panel changes include:

1. Toggling visibility of the legend
2. Changing colors via the legend
3. Toggling visibility of the panel title
4. Changing the panel title

I think it might be easier to re-arrange this section to just cover those topics as part of the tutorial, and then mention that it's separate from editing the visualization.

Reply:
Revised.

Reply:
@joshuarrrr Will you review this revised section? Do you think we should have a section herein on editing visualizations? Or do you think that tutorial would be better suited to the Building data visualizations section?